cmd/compose/run.go

```diff
@@ -141,6 +142,7 @@ func runCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.Detach, "detach", "d", false, "Run container in background and print container ID")
 	flags.StringArrayVarP(&opts.environment, "env", "e", []string{}, "Set environment variables")
+	flags.StringArrayVar(&opts.EnvFiles, "env-file", []string{}, "Read in a file of environment variables")
```

Comment:
I suggest we use `--service-env-file` to avoid ambiguity with upper-level compose command --env-file flag

Reply:
If we want to retain a parallel between `docker run` and `docker compose run`, then I'm wondering if we should look at doing the reverse, and think of a new name for the flag for setting the `.env`  (we could keep the old flag, but hidden)